showcases/showcase-styles.css

```
@@ -12,15 +12,24 @@
 	padding: var(--db-spacing-fixed-md);
 }
 
+.variants-card {
+	margin: var(--db-spacing-fixed-md);
+}
+
 .variants-list {
-	list-style-type: none;
-	margin: 0;
-	padding: 0;
 	display: flex;
 	flex-wrap: wrap;
 	gap: var(--db-spacing-fixed-md);
 }
 
+dd {
```

```suggestion
.variants-list dd {
```

This would be quite a global selector.